src/main/java/lotto/domain/RandomNumbersGenerator.java

```diff
@@ -9,21 +9,20 @@
 public class RandomNumbersGenerator {
```

Comment:
```suggestion
public final class RandomNumbersGenerator {
```
private 생성자 추가한 김에 클래스에 `final` 키워드까지도 추가해볼 수 있겠네요